app/sessionStore.js

```diff
@@ -626,7 +628,8 @@ module.exports.defaultAppState = () => {
       pendingRecords: {}
     },
     locationSiteKeysCache: undefined,
-    sites: getTopSiteMap(),
+    sites: getTopSiteMap(), // TODO remove
```

Comment:
what will happen to the top sites? currently they have siteTag DEFAULT so that they don't show up in history/autosuggest until they are actually visited